src/components/Box/Box.tsx

```diff
@@ -0,0 +1,16 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from 'react';
+
+interface DataProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
We don't use Interface in Fondue. You can set the type like this instead and also DataProps is a misleading name.
```
export type BoxProps = {
    'data-test-id'?: string;
} & React.HTMLAttributes<HTMLDivElement>;
```